bridge/engine/EngineMixer.cpp

```diff
@@ -3891,4 +3941,23 @@ void EngineMixer::onBarbellUserMediaMap(size_t barbellIdHash, const char* messag
     }
 }
 
+void EngineMixer::onBarbellMinUplinkEstimate(size_t barbellIdHash, const char* message)
+{
+    auto it = _engineBarbells.find(barbellIdHash);
+    if (it == _engineBarbells.end())
+    {
+        return;
+    }
+
+    EngineBarbell& barbell = *(it->second);
+
+    logger::info("received BB msg over barbell %s %zu, json %s",
```

Comment:
this would be quite some spam. debug level perhaps.
And it is specifically minClientDownlinkBandwidth

Reply:
Changed to debug level. What did yo mean "specifically..."? We'll see JSON in the log.